runtime/src/bank.rs

```diff
@@ -6525,6 +6540,15 @@ impl Bank {
             );
         }
 
+        if new_feature_activations.contains(&feature_set::raise_block_limits_to_60m::id()) {
```

Comment:
not sure exactly how we want to handle this conditional logic.. E.g. what would we want to happen if both features were activated in the same epoch?

Currently, we will write the 207 limits then the 256 limits. That seems right to me. Practically speaking, we will activate these in separate epochs, of course.

same thing applies to `fn finish_init` above

Reply:
intent is for 207, then 256. the ordering here to me makes sense; if 256 is active, it takes priority over 207, regardless of the actual status of 207.

Reply:
we should have a test for that case as well? activate 256 first, then 207, make sure we have 256 limits.

Reply:
test added